### ISSUE TITLE
SyntaxError: Cannot use import statement outside a module

### ISSUE THREAD

**Issue by smithki** [aveq-research/localforage-asyncstorage-driver#1]
I am encountering the above `SyntaxError` when running [`ava`](https://github.com/avajs/ava) tests against files which utilize this module. I am using TypeScript + [`ts-node`](https://www.npmjs.com/package/ts-node), which is probably why it complains (see https://github.com/TypeStrong/ts-node/issues/505).

Would it be possible for this module to use a CommonJS `require` instead of `import`?

**Comment by smithki** [comment]
Here's my current workaround (using [`mockery`](https://github.com/mfncooper/mockery)) until a fix is implemented:

```ts
mockery.registerMock('@aveq-research/localforage-asyncstorage-driver', {});
```

**Comment by slhck** [comment]
You're right, the NodeJS library code should use `require`.

I would instead suggest to compile to CommonJS with Babel, as we'd like to use ES6 imports in the original code base.

I've created a PR at #2 that should fix it, but I will wait for @fohletex to check it.

**Comment by smithki** [comment]
Awesome turnaround time on a PR 👏  looking forward to seeing this release! Thanks for building this helpful module!

**Comment by fohletex** [comment]
Thanks @smithki for this issue. I merged #2 and made a new release to v1.1.0

Have fun! :)
